### ISSUE TITLE
It seems it cannot run on the GPU（4060ti）

### ISSUE THREAD

**Issue by laugh-xixi** [jrzaurin/pytorch-widedeep#243]
tab_trainer = Trainer(  
                model=tab_model,  
                objective="binary",  
                optimizers=torch.optim.Adam(tab_model.parameters(), lr=0.001),  
                metrics=[Accuracy, Precision, F1Score, Recall],  
                seed=random_state_tmp,  # 使用相同的随机种子 
                device="cuda" 
            )  
I selected CUDA for the device here, but it still runs slowly.GPU usage shows 2%
